astroid/exceptions.py

```diff
@@ -272,6 +272,24 @@ def __init__(self, target: "nodes.NodeNG") -> None:
         super().__init__(message=f"Parent not found on {target!r}.")
 
 
+class StatementMissing(ParentMissingError):
+    """Raised when a call to node.statement() does not return a node. This is because
+    a node in the chain does not have a parent attribute and therefore does not
+    return a node for statement().
+
+    Standard attributes:
+        target: The node for which the parent lookup failed.
+    """
+
+    def __init__(self, target: "nodes.NodeNG") -> None:
+        # See: https://github.com/PyCQA/pylint/issues/2903
+        # and: https://github.com/PyCQA/astroid/pull/1217#discussion_r744149027
+        # pylint: disable-next=bad-super-call
```

Comment:
```suggestion
        # pylint: disable-next=bad-super-call
        # https://github.com/PyCQA/pylint/issues/2903
        # https://github.com/PyCQA/astroid/pull/1217#discussion_r744149027
```
That's usually how I do it.

Reply:
Oh wow, does this work?
That's good to know for https://github.com/PyCQA/pylint/issues/5258

Reply:
Oh wow, does this work?
That's good to know for https://github.com/PyCQA/pylint/issues/5258